### PR TITLE
Update swagger ( timestamps for CRUD)

### PR DESCRIPTION
**Pull request by przyjacielpkp**

### CODE REVIEW

File: swagger.yaml [hunk 1]
```diff
@@ -431,7 +431,7 @@ components:
           example: sample_project
         created_at:
           type: integer
-          format: timestamp
+          format: string
           example: 1
         last_modified:
           type: integer
```
Comment (ghost) [inline swagger.yaml]: Wait, so those are integers or string?

**Review by ghost — CHANGES_REQUESTED** [review]

File: swagger.yaml (hunk 1 above)
Comment (przyjacielpkp) [reply swagger.yaml]: i handle them as strings ( that means all the conversion is done on the server side, using serialize/deserialize methods visible in the second PR), do we want to have them as integers (int64) anyway?

**Review by ghost — APPROVED** [review]

File: swagger.yaml (hunk 1 above)
Comment (ghost) [reply swagger.yaml]: No, it is fine this way.

**Comment by przyjacielpkp** [comment]
I've updated this swagger once again, so you can check if everything's still fine

**Review by ghost — APPROVED** [review]
Looks fine

**Review by ghost — CHANGES_REQUESTED** [review]
Wait a sec, shouldn't there be `owner_id` AND `owner_email`?

**Comment by przyjacielpkp** [comment]
You intend to use `user_id` if you have `user_email`?

**Comment by ghost** [comment]
> You intend to use `user_id` if you have `user_email`?

Both might be needed

**Comment by przyjacielpkp** [comment]
oh boy, alright ,then gimme a second

**Review by ghost — APPROVED** [review]

**Pull request merged**